cmake/ctest/drivers/atdm/README.md

```diff
@@ -441,16 +441,29 @@ The following `<system_name>` sub-directories exist (in alphabetical order):
 
 ## How to add a new system
 
-To add a new system, first add a new `elseif` statement for the new system in
-the file:
+To add a new system, first the new system to the file:
```

Comment:
To add a new system, first the new system must be added to the file:

Reply:
Thanks I have fixed the wording to hopefully be a little better.